packages/ui/src/council/components/PastVotes/PastVote/PastVote.tsx

```diff
@@ -39,8 +41,9 @@ export const PastVote = ({ vote, latestCycleId, $colLayout }: PastVoteProps) =>
   }
 
   // Reflects if the vote was cast in latest election
-  const isLatestElection = vote.cycleId === latestCycleId
-  const isVoteStakeLocked = useIsVoteStakeLocked(vote.voteFor, { isLatestElection })
+  const isVoteStakeLocked = useGroupLocks(vote.castBy, balance?.locks ?? []).unRecoverable.some(
+    (lock) => lock.type === 'Voting'
+  )
```

Comment:
Here I think a `useIsVoteLockRecoverable(hasVoteLock, vote.castBy)` should work if it's exported by: `@/accounts/hooks/useGroupLocks`